app/views/bgeigie_imports/_auto_approve_status.html.erb

```diff
@@ -0,0 +1,23 @@
+<ul class="unstyled nav" id="progress" >
+  <li class="nav-header">
+    <%= "Status" %>
+  </li>
+  <li>
+    <%- if @bgeigie_import.auto_apprv_no_zero_cpm -%>
```

Comment:
You might want to create view helper method to simplfy. For example,

```erb
<%= t('.no_cpm_equals_zero') %>
<%= cpu_equals_zero_icon(@bgeigie_import) %>
```

and, in view helper,

```ruby
def cpu_equals_zero_icon(import)
  content_tag(:i, class: import.auto_apprv_no_zero_cpm ? 'icon-ok pull-right' : 'icon-remove pull-right')
end
```

Reply:
For example, https://github.com/Safecast/safecastapi/pull/587/commits/ebd002df4a7507f7dfae776d22eb0f71d3df20e9